### PR TITLE
Changed nt to n_tok, to better conform with style guidelines

### PR DESCRIPTION
**Pull request by duckysherwood**
The style guidelines say that the count of something should be `n_` and not `n`; the abbreviations say that `t` is tensor, so I called it `tok` instead (which is used elsewhere).  Also fixed a comment.

Also updated `abbr.md` to add `tok`.

### CODE REVIEW

File: docs/abbr.md [hunk 1]
```diff
@@ -152,3 +152,4 @@ Note that there are always exceptions, especially when we try to comply with the
 |                  | activation                          | actn           |                                                  |
 |                  |                                     |                |                                                  |
 | **Text**         | string                              | s              |                                                  |
+|                  | token                               | tok            |                                                  |
```
Comment (apiltamang) [inline docs/abbr.md]: did you mean this to be 'n_tok' instead of 'tok'? I could be mistaken ...

Comment (jph00) [reply docs/abbr.md]: No `tok` is correct here

Comment (duckysherwood) [reply docs/abbr.md]: No, there is an entry on line 30 defining the prefix `n_`.  So this like defines `tok` for `token` and `n_tok` is the number of tokens.

**Comment by jph00** [comment]
Much clearer - thanks!

**Pull request merged**